### PR TITLE
Add setting to blur background during gameplay

### LINKED ISSUE #1736: Add setting to blur background during gameplay
For people who think 100% background dim is too dark, but still don't want to get distracted by the background. It could be listed in the settings under background dim.
I've seen this used in a couple of places already (like the song select screen), so it would just be a matter of reusing that code and adding a toggle to settings.

### PR DESCRIPTION
**Pull request by thatprogrammer1**
Closes #1736

### CODE REVIEW

File: osu.Game/Configuration/OsuConfigManager.cs [hunk 1]
```diff
@@ -65,6 +65,7 @@ protected override void InitialiseDefaults()
 
             // Gameplay
             Set(OsuSetting.DimLevel, 0.3, 0, 1, 0.01);
+            Set(OsuSetting.BlurLevel, false);
```
Comment (smoogipoo) [inline osu.Game/Configuration/OsuConfigManager.cs]: Either make this variable similar to `BlurLevel` or rename it to `GameplayBackgroundBlur`.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
also was this not just a copy-paste of @ColdVolcano's PR?

**Comment by peppy** [comment]
if so we should close this and use that one.

**Comment by smoogipoo** [comment]
Keep in mind that they cannot commit to ColdVolcano's PR.

**Comment by smoogipoo** [comment]
Let's close this one and work on ColdVolcano's.

**Pull request closed without merging**